src/lib/settings/GatewayDriver.js

```diff
@@ -63,29 +62,29 @@ class GatewayDriver {
 		return {
```

Comment:
This could probably be moved as well, but that would require it to be a function and I'm not sure that's the best idea in constants.

Reply:
They're not constants but variables, since they depend on `client`. That's why I decided to put there. Nonetheless this can be changed later if we find a better way.